app/common/util/dateUtil.js

```diff
@@ -65,6 +65,13 @@ Bahmni.Common.Util.DateUtil = {
     getDateWithoutTime: function(datetime){
         return datetime?moment(datetime).format("YYYY-MM-DD"):null;
     },
+    
+    getDateInMonthsAndYears : function (date, format){
+        var format = format || "MMM YY";
```

Comment:
@valvijo I think line 70 is supposed to be  format = format || "MMM YY" without the var keyword because it is already declared in line 69 i.e getDateInMonthsAndYears : function (date, format)